constants/MissionsConstants.js

```diff
@@ -0,0 +1,55 @@
+import ICONS from "./icons"
+
+const MISSIONSCONSTANTS = {
```

Comment:
Since this is mock data given that the data ideally should come from our database, you can move this file to a new folder which you'll create on the root of your project called mockData